notary-commons/src/main/kotlin/com/d3/commons/service/FInalizationDetails.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright D3 Ledger, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.d3.commons.service
+
+import com.google.gson.Gson
+import java.math.BigDecimal
+
+private val gson = Gson()
+
+/**
+ * Data class that stores details of withdrawal finalization
+ * @param withdrawalAmount - amount of withdrawal operation
+ * @param withdrawalAssetId - asset id of withdrawal
+ * @param feeAmount - amount of fee for withdrawal
+ * @param feeAssetId - asset id of fee
+ * @param srcAccountId - id of account that initiated current withdrawal
+ * @param withdrawalTime - time of withdrawal
+ */
+data class FinalizationDetails(
```

Comment:
how about renaming to `WithdrawalFinalizationDetails` since it's about withdrawals